src/NServiceBus.Core/Audit/AuditConfig.cs

```diff
@@ -12,6 +12,10 @@ public class AuditConfig : ConfigurationSection
         /// Gets/sets the address to which messages received will be forwarded.
         /// </summary>
         [ConfigurationProperty("QueueName", IsRequired = false)]
+        [ObsoleteEx(
+        Message = "Audit configuration is no longer available via configuration files. Use EndpointConfiguration.AuditProcessedMessagesTo()",
```

Comment:
~~files~~ file?

I think we can leave out the `Use EndpointConfiguration.AuditProcessedMessagesTo()` part, as you specified this as the `ReplacementTypeOrMember`.